src/swapd/swap_state.rs

```diff
@@ -541,7 +541,10 @@ impl StateMachine<Runtime, Error> for SwapStateMachine {
             }
             SwapStateMachine::AlicePunish => try_alice_punish_to_swap_end(event, runtime),
 
-            _ => Ok(Some(self)),
+            SwapStateMachine::SwapEnd(_) => {
+                runtime.log_warn("Received event after swap end");
```

Comment:
`warn` level seems a bit much. There are bound to be some incoming events sometimes. I'd rather return `None` instead and let the executor log on `debug` level.

Reply:
b1469847